nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Kindly ensure your correct_name method raises `NotImplementedError` and also a string that outputs specifics of the error raised.

`hint`
`raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`